File: server/Responses/Municipio.cs
namespace server.Responses
{
    public class MunicipioRes : BaseRes
    {
        public required int Id { get; set; }
        public int IdProvincia { get; set; }
        public required string Nombre { get; set; }
        public required string Provincia { get; set; }

    }
}

File: web/src/views/private/Municipio/validations/municipio.ts
import * as Yup from "yup";

export const municipioSchema = Yup.object({
  nombre: Yup.string().required("Nombre es requerido"),
  idProvincia: Yup.number().required("La provincia es requerida"),
});

export interface MunicipioForm extends Yup.InferType<typeof municipioSchema> {}
